### ISSUE TITLE
Keyboard focus should be in terminal when launched

### ISSUE THREAD

**Issue by illume** [illume/headlamp#43]
When I launch a terminal in a pod, I need to click on the terminal window before I can type commands. If I type commands before clicking in the terminal, nothing happens. Every time a user launches a terminal, they intend on typing commands into it.

As soon as the terminal is launched, the keyboard focus moves to it.

## What users will benefit from this feature?

This will save a click every time someone launches a terminal. It'll also be a less confusing experience for new users who will need to figure out they need to click into the terminal before they can type commands.
